### PR TITLE
Add systemctl support

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/process/ChildProcessImpl.java [hunk 1]
```diff
@@ -34,49 +45,63 @@ public String commandLine() {
...
+            throw new CommandException(message);
         }
 
         return this;
     }
 
-    private String suffixOfOutputForLog() {
-        String output = getUtf8Output();
+    @Override
+    public UnexpectedOutputException newUnexpectedOutputException(String problem) {
```
Comment (freva) [inline node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/process/ChildProcessImpl.java]: Why is this a method? Think this should just be done in the constructor of the exception.

**Review by freva — APPROVED** [review]

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/process/ChildProcessImpl.java (hunk 1 above)
Comment (hakonhall) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/process/ChildProcessImpl.java]: The method needs access to commandLine() and getUtf8Output() from this instance. I could make a constructing taking 'problem', and those 2 arguments?

Comment (freva) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/process/ChildProcessImpl.java]: Sounds good.

**Review by freva — APPROVED** [review]

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/process/ChildProcessImpl.java (hunk 1 above)
Comment (hakonhall) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/process/ChildProcessImpl.java]: Done

**Pull request merged**